sphinx_scality/static/css/current.css_t

```diff
@@ -265,3 +265,8 @@ div.footer div.rightwrapper {
 div.footer div.rightwrapper img.social {
     margin: auto 15px;
 }
+
+.term {
+    color: #F37533;
```

Comment:
Not sure exactly what a `.term` is, but:

- It probably doesn't belong among footer styles
- Colours shouldn't be hardcoded if possible, unless it doesn't make sense to change them when using a different colour palette (you can get inspiration from other uses of `{{ theme_<something>color }}` and the corresponding definition in `theme.conf`)